Remotion/Web/Core/Res/Themes/NovaViso/HTML/WebButton.css

```diff
@@ -2,108 +2,55 @@
 
 button.webButton
 {
-  background-color: Transparent;
-  cursor: hand;
+  background-color: transparent;
+  border: var(--remotion-themed-border-width) solid var(--color-primary);
+  border-radius: var(--remotion-themed-border-radius);
+  padding: 0.4em 1.2em;
+  box-sizing: border-box;
   cursor: pointer;
-  text-decoration: none !important;
-  border: none;
-  position: relative;
-  display: inline-block;
-  vertical-align: middle;
-  padding: 0px 2px;
-  -moz-padding-start: 0;
-  -moz-padding-end: 0;
-  outline: none;
-}
-
-button.webButton::-moz-focus-inner
-{ 
-  border: 0;
 }
 
-button.webButton span.buttonBody
+button.webButton:hover,
+button.webButton:focus
 {
-  border: solid 1px #AAAAAA;
-  font-weight: bold;
-  background-image: url(../Image/sprite.svg#ButtonBackground);
-  background-size: 1px 20px;
-  background-color: #E6E5E7;
-  background-position: center top;
-  background-repeat: repeat-x;
-  
-  display: block;
-  position: relative;
-  top: 0px;
-  left: -2px;
-  
-  width: 100%;
-  white-space: nowrap;
-
-  padding-top: 2px;
-  padding-bottom: 2px;
-  padding-left: 1px;
-  padding-right: 1px;
+  background-color: var(--color-highlight);
 }
 
-button.webButton span.buttonBody:first-child::before
+button.webButton[disabled]
 {
-  content: "";
-  height: 1.25em;
-  width: 0;
-  display: inline-block;
-  vertical-align: text-bottom;
-}
-
-button.webButton span.buttonBody img
-{
-  vertical-align: bottom;
-  border: none;
+  background-color: transparent;
+  color: var(--color-standard-border);
+  border: var(--remotion-themed-border);
+  cursor: unset;
 }
 
-button.webButton:focus span.buttonBody,
-button.webButton:hover span.buttonBody
-{
-  background-image: url(../Image/sprite.svg#ButtonBackgroundHover);
-  background-size: 1px 20px;
-  background-color: #DEECFC;
-  border: 1px solid #7CB2E6;
-}
-
-button.webButton:focus span.buttonBody
+button.webButton span.buttonBody
 {
-  text-decoration: underline;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
-button.webButton:active span.buttonBody
+button.webButton[disabled] span.buttonBody img
 {
-  background-image: url(../Image/sprite.svg#ButtonBackgroundActive);
-  background-size: 1px 20px;
-  background-color: #C5E0FC;
-  border: 1px solid #569CE0;
+  opacity: 0.5;
+  filter: grayscale(100%);
```

Comment:
disabled images will recur in the styles. should we make common properties and/or do a re-motion themed option? Basically, we could go with re-motion themed image that is nested inside parent that has either the [disabled] attribute or the `.disabled` class applied.